features/partners.feature

```diff
@@ -153,378 +256,229 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
-      | naam         | waarde |
-      | code         | M      |
-      | omschrijving | man    |
-      En de partner heeft de volgende naam gegevens
+      | burgerservicenummer |           |
+      En de partner heeft GEEN 'geslachtsaanduiding' gegevens
+      En de partner heeft alleen de volgende 'naam' gegevens
       | naam                                 | waarde    |
-      | voornamen                            | Cees      |
-      | voorvoegsel                          | de        |
-      | geslachtsnaam                        | Vries     |
-      En de partner heeft GEEN ontbindingHuwelijkPartnerschap gegevens
+      | geslachtsnaam                        | .         |
+      En de partner heeft alleen de volgende 'geboorte' gegevens
+      | naam                | waarde    |
+      | datum               | 00000000  |
+      En de partner heeft de volgende 'soortVerbintenis' gegevens
+      | naam         | waarde   |
+      | code         | .        |
+      En de partner heeft alleen de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      | naam                | waarde    |
+      | datum               | 19560327  |      
+      En de partner met burgerservicenummer '555550005' heeft GEEN ontbindingHuwelijkPartnerschap gegevens
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                                                    |
-      | type                | RaadpleegMetBurgerservicenummer                           |
-      | burgerservicenummer | 999992807                                                 |
-      | fields              | naam,partners.naam.voornamen,partners.burgerservicenummer |
-      En heeft de persoon met burgerservicenummer '999992807' alleen 'partners' met de volgende gegevens
-      | burgerservicenummer |
-      | 999992808           |
-      En heeft de partner met burgerservicenummer '999992807' alleen de volgende 'naam' gegevens
-      | naam                                 | waarde    |
-      | voornamen                            | Cees      |
-      En heeft de partner geen andere gegevens dan 'naam' en 'burgerservicenummer'
-
-  TODO: bepalen OnbekendPartner
-    @proxy
-    Scenario: Partner is volledig onbekend
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999993008                       |
+      | fields              | partners                        |
+      Dan heeft de persoon met burgerservicenummer '999993008' exact 1 'partners'
+      En heeft de partner alleen de volgende gegevens
+      | naam              | waarde          |
+      | type              | OnbekendPartner |
+      | indicatieOnbekend | true            |
 
     @proxy
     Scenario: Met fields zijn alleen velden zonder waarde gevraagd
+      Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
Dit is toch een "fields-feature" scenario ? Dit is niet specifiek voor partners.

Reply:
ik bedoel hiermee te tonen dat gebruikte fields geen invloed heeft op het type van de partner. Hoe zou ik dit scenario moeten noemen om dat wel duidelijk te maken?

Reply:
Je zou hiervoor een rule kunnen definieren: 'type property wordt altijd geretourneerd'

- fields bevat alleen de naam van een property zonder waarde
- fields bevat de naam van een property met waarde
- fields bevat alleen de naam van de 'type' property

Reply:
Staat inmiddels in de fields-feature.